src/keyboard/KeymapBindingChildren.tsx

```diff
@@ -0,0 +1,71 @@
+import { HidUsageLabel } from "./HidUsageLabel";
+
+export interface KeyBinding {
+  param1: number;
+  param2: number;
+}
+
+const renderIndicator = (numbers: 0|1|2|3 = 1): JSX.Element => (
+    <div className="absolute ml-0.5 h-full flex flex-col justify-evenly py-1">
+      {Array.from({ length: Math.max(1, numbers) }, (_, i) => (
+        <div 
+          key={i}
+          className="bg-gray-900 opacity-80 rounded-full" 
+          style={{
+            width: '.2rem',
+            height: numbers === 0 ? '0.8rem' : '.2rem',
+          }}
+        />
+      ))}
+    </div>
+  );
+
+export const getBindingChildren = (
+  behavior: string,
+  binding: KeyBinding,
+): JSX.Element | JSX.Element[] => {
+  if (behavior === "Layer-Tap") {
```

Comment:
Those if should probably be improved, not sure if we should add the "behavior type" and/or binding-cells. Could maybe be a followup?